utils/frontend/microapps-pages/LiqPools.ts

```diff
@@ -201,4 +203,19 @@ export class LiqPools {
       timeout,
     );
   }
+
+  async getPoolsList() {
+    const classNameXpath = await buildClassXpath("focus:outline-0 group");
```

Comment:
FYI - in future we can add datatesid locators directly in Microapps repo and send PR to Matus/Dominik. But thats just info for future :)

Reply:
I'll try not to forget about it)